guarddog/scanners/__init__.py

```diff
@@ -0,0 +1,21 @@
+from .pypi_package_scanner import PypiPackageScanner
+from .pypi_project_scanner import PypiRequirementsScanner
+from .npm_package_scanner import NPMPackageScanner
+from .npm_project_scanner import NPMRequirementsScanner
+
+from typing import Optional
+from .scanner import Scanner
+from ..ecosystems import ECOSYSTEM
+
+
+def get_scanner(ecosystem: ECOSYSTEM, project: bool) -> Optional[Scanner]:
```

Comment:
about the discussion in https://github.com/DataDog/guarddog/pull/101/files#diff-9d026416008f88615d7f288b80a67f0e2ede3219b3aad8ed9c7aece17cf6c8c3

changing to the suggested approach would also allow to get rid of this `project` boolean which IMHO is confusing

Reply:
+1

Reply:
Let's keep this one for another cycle (as it is mostly internal)